packages/invoices/src/InboundShipment/DetailView/DetailView.tsx

```diff
@@ -18,8 +18,8 @@ import { AppBarButtons } from './AppBarButtons';
 import { SidePanel } from './SidePanel';
 import { GeneralTab } from './GeneralTab';
 import { InboundLineEdit } from './modals/InboundLineEdit';
-import { InvoiceLine, InboundShipmentItem } from '../../types';
-import { useInbound } from '../api';
+import { InboundItem } from '../../types';
```

Comment:
Same as outbound, removing use of `InvoiceLine`, renaming `InboundShipmentItem` to `InboundItem`